runtime/tutor/tutor.tr.iso9

```diff
@@ -1,6 +1,6 @@
-===============================================================================
-=    V I M   T u t o r ' a   h o �   g e l d i n i z !   -     S�r�m 1.7      =
-===============================================================================
+================================================================================
+=     V I M   T u t o r ' a   h o �   g e l d i n i z !    --    S�r�m 1.7     =
+================================================================================
```

Comment:
English version of tutor is 79 columns here. I think it's better to keep the same width here.

Reply:
I added that extra column to make it look nicer and equal on both sides. It's up to you, but if there is not a specific reason for this it would be better this way from a visual standpoint.